addons/common/functions/fnc_addToInventory.sqf

```diff
@@ -130,7 +130,7 @@ switch (_type select 0) do {
             private _pos = _unit modelToWorldVisual [0,1,0.05];
 
             _unit = createVehicle ["WeaponHolder_Single_F", _pos, [], 0, "NONE"];
```

Comment:
Do we want to change this, so that it searched for existing weapon holders before creating a new one?
If this is used in a loop on a unit that has no space, it will created multiple weapon holders unnecessarily.

Reply:
Not a bad idea, but I think that can be a separate PR